api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/MedicationTranslatorImpl.java

```diff
@@ -47,10 +47,9 @@ public Medication toFhirResource(Drug drug) {
 		medication.setCode(conceptTranslator.toFhirResource(drug.getConcept()));
 		medication.setForm(conceptTranslator.toFhirResource(drug.getDosageForm()));
 		
-		Medication.MedicationIngredientComponent ingredient = new Medication.MedicationIngredientComponent();
-		CodeableConcept codeableConcept;
 		for (DrugIngredient val : drug.getIngredients()) {
-			codeableConcept = conceptTranslator.toFhirResource(val.getIngredient());
+			Medication.MedicationIngredientComponent ingredient = new Medication.MedicationIngredientComponent();
+			CodeableConcept codeableConcept = conceptTranslator.toFhirResource(val.getIngredient());
 			if (val.getStrength() != null) {
 				codeableConcept.setText(val.getStrength().toString());
```

Comment:
This should probably map to `ingredient.strength`. If that won't work, we should just leave it out. This is definitely the wrong place to map the strength.

Reply:
Noted